### PR TITLE
improvement: Privilege to change default breakpoint of tablet & mobile device

### PR DESCRIPTION
**Pull request by Balachandark**

### CODE REVIEW

File: inc/core/common-functions.php [hunk 1]
```diff
@@ -1316,3 +1316,48 @@ function astra_get_db_option( $option, $default = '', $deprecated = '' ) {
 	 */
 	return apply_filters( "astra_get_db_option_{$option}", $value, $option, $default );
 }
+
+/**
+ * Get the tablet breakpoint value.
+ *
```
Comment (premanshup) [inline inc/core/common-functions.php]: can we add since comment here?

File: inc/core/common-functions.php [hunk 2]
```diff
@@ -1316,3 +1316,48 @@ function astra_get_db_option( $option, $default = '', $deprecated = '' ) {
...
+		$header_breakpoint = $header_breakpoint + $max;
+	}
+
+	return absint( $header_breakpoint );
+}
+
+/**
+ * Get the mobile breakpoint value.
+ *
+ * @param string $min min.
```
Comment (premanshup) [inline inc/core/common-functions.php]: can we add since comment here?

File: inc/core/common-functions.php (hunk 1 above)
Comment (Balachandark) [reply inc/core/common-functions.php]: Since comment added

File: inc/compatibility/edd/class-astra-edd.php [hunk 3]
```diff
@@ -390,6 +390,93 @@ public function add_styles( $assets ) {
 		 */
 		public function add_inline_scripts() {
 
+			$edd_css_output = '';
```
Comment (premanshup) [inline inc/compatibility/edd/class-astra-edd.php]: @imnavanath Can we remove this and directly add it `$edd_css_output .= ` to `$edd_css_output = ` for the first occerence.

Comment (imnavanath) [reply inc/compatibility/edd/class-astra-edd.php]: Corrected here - https://github.com/brainstormforce/astra/pull/1533/commits/22d60d967a10ea0fb498c92fe5aaea3ed932c647

**Pull request merged**